content/rest/channels.textile

```diff
@@ -18,6 +18,8 @@ jump_to:
   Help with:
     - Getting started#getting-started
     - Channels#channels
+    - Obtaining history of a channel#channel-history
```

Comment:
@tomczoink I am afraid you need to add `#publishing` here as it's missing

Reply:
Also ,why is this obtaining history and getting history below?

Reply:
Fixed both with https://github.com/ably/docs/commit/f0f9e2a19363fcb157ec7fde54dea03b1244f3ef.